STYLE_GUIDE.md

```diff
@@ -154,6 +154,27 @@ Prefer:
 
 ## JavaScript
 
+### Variable naming
+
+- Prefer camelCase over snake_case, except in JSON.
+- Try to use verbs for function/method names, and nouns for constant values.
+- Boolean variable names should give some indication of their state.
+- Prefer clarity over brevity, and avoid single-letter names and abbreviations unless they are well known for this domain (e.g. JSON).
+
+Avoid:
+
+```js
+const actNode = true;
+const nodeState = (d) => actNode && d.node;
+```
+
+Prefer:
+
+```js
+const isActiveNode = true;
+const getNodeState = (state) => isActiveNode && state.node;
+```
+
```

Comment:
Might be worth ending with a link to the [AirBnb JS guide naming conventions](https://github.com/airbnb/javascript#naming-conventions)?

Reply:
Not a bad suggestion. I couldn't find a way to slot it in neatly so I just tried to ensure that I covered all the same broad points.